common/app/model/VideoPlayer.scala

```diff
@@ -16,6 +16,7 @@ case class VideoPlayer(
   title: String,
   autoPlay: Boolean,
   showControlsAtStart: Boolean,
+  // TODO make Option[String] as relies on section being set
```

Comment:
if there isn't a section `endSlatePath` will 404 - it shouldn't be on the DOM in this case